src/phasorpy/flim.py

```diff
@@ -0,0 +1,238 @@
+"""Manipulate Fluorescence Lifetime Microscopy (FLIM) data for phasor analysis.
+
+The ``phasorpy.flim`` module provides functions for the calibration of time-
+resolved fluorescence lifetime imaging (FLIM) data using phasor techniques. The
+module is designed to facilitate the processing and interpretation of FLIM data,
```

Comment:
Sorry I missed that line. Do you have a tool to automatically format the docstring elements to limit them to 79 characters? Because black won't cover docstrings and I tried blackdoc and docformatter but to no avail, in this case at least. Should we check all the docstring lines manually to be below the 79 characters limit?

Reply:
`flake8` caught this. Anyway, there are cases where docstrings may exceed the maximum line lengths, for example, for long URLs (https://github.com/phasorpy/phasorpy/blob/6614090d13e91ff407d4c4db45aa6aa4d8a65b01/src/phasorpy/datasets.py#L14C1-L14C108). Writing numpy docstrings is tedious. Can you try [ruff](https://docs.astral.sh/ruff/faq/#does-ruff-support-numpy-or-google-style-docstrings). Ultimately we would like to replace all linters (isort, black, pylint...) with ruff.

Reply:
I tried ruff, but it didn't catch those issues. Now with flake8 I tried in worked perfectly to check for lines that exceeded the limit. Maybe we can think of implementing some test to enforce this or add the check in the contributors guide.